### ISSUE TITLE
Something wrong with single.b.spa.R in data/Makefile.in

### ISSUE THREAD

**Issue by bmcgough** [statgen/EPACTS#3]
I think single.b.spa.R does not exist, and single.b.spa2.R,single.b.spa3.R do exist.

This is causing a make failure on Ubuntu 14.04 and 16.04 for me.

**Comment by hyunminkang** [comment]
That is correct. There was a SPA test before, which had a bug, and it stayed for a while and the bug gave confusions. So when we created the github repository, we named it "spa2", and updated the wiki page too, just to avoid the confusion between old vs. new version. So we will not have "SPA" test. As written in the wiki page, spa2 test is the official test to use (and sna2 test is a faster version of it, which is not currently documented)
